glide.yaml

```diff
@@ -1,11 +1,11 @@
 package: github.com/decred/dcrd
 import:
 - package: github.com/agl/ed25519
-  version: 278e1ec8e8a6e017cd07577924d6766039146ced
   subpackages:
   - edwards25519
 - package: github.com/btcsuite/btclog
-- package: github.com/btcsuite/go-flags
+- package: github.com/jessevdk/go-flags
+  version: ^1.3.0
 - package: github.com/btcsuite/go-socks
```

Comment:
the btcsuite fork of go-flags should be removed since we are switching to upstream.

Reply:
commented on wrong file, it might be a transient dep in case we need to figure out what imported package is requiring it.